src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -39,10 +39,10 @@ azure-mgmt-datalake-nspkg==3.0.1
 azure-mgmt-datalake-store==1.1.0b1
 azure-mgmt-datamigration==10.0.0
 azure-mgmt-dns==8.0.0
-azure-mgmt-eventgrid==10.2.0b2
+azure-mgmt-eventgrid==10.2.0
```

Comment:
It seems there was a breaking change between `azure-mgmt-eventgrid==10.2.0b2` and `azure-mgmt-eventgrid==10.2.0`. That breaks the whole idea of semantic versioning. There should be no API changes anymore, once you get to beta releases.

```
ImportError: cannot import name 'ResourceSku' from 'azure.mgmt.eventgrid.models' (/home/runner/work/azure-cli/azure-cli/env/lib/python3.12/site-packages/azure/mgmt/eventgrid/models/__init__.py). Did you mean: 'Resource'?
```

Reply:
Moved into separate issue https://github.com/Azure/azure-cli/issues/31520